src/Command/MigrateRosalieIdsToSiSiaoIdsCommand.php

```diff
@@ -76,7 +77,8 @@ private function updateBeneficiary(Beneficiaire $beneficiary, array $records, Cl
     {
         if ($externalLink = $beneficiary->getExternalLinkForClient($rosalieClient)) {
             $siSiaoNumber = $records[$externalLink->getDistantId()] ?? null;
-            $externalLink->setDistantId($siSiaoNumber);
+            $newExternalLink = ClientBeneficiaire::createForMember($rosalieClient, $siSiaoNumber, $externalLink->getMembreDistantId());
+            $beneficiary->addExternalLink($newExternalLink);
```

Comment:
I don't think we need the $newExternalLink var
`$beneficiary->addExternalLink(ClientBeneficiaire::createForMember($rosalieClient, $siSiaoNumber, $externalLink->getMembreDistantId()));`